src/core/DirectGeometry.js

```diff
@@ -34,7 +35,7 @@ function DirectGeometry() {
 
 }
 
-Object.assign( DirectGeometry.prototype, {
+DirectGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ), {
```

Comment:
It seems missing EventDispatcher inherit. I added.

Reply:
Um, I don't think this inheritance is needed. No `EventDispatcher` method is used.

Reply:
So removing the inheritance from JS and TS?

Reply:
Removed